mitiq/ddd/insertion.py

```diff
@@ -80,3 +82,52 @@ def get_slack_matrix_from_circuit_mask(mask: np.ndarray) -> np.ndarray:
                         break
 
     return slack_matrix
+
+
+def _construct_replacements(
+    circuit: Circuit, sequence: Circuit, index: int
+) -> List[Tuple[int, Operation, Operation]]:
+    """Returns the replacements for insert_ddd_sequences."""
+
+    return [
+        (index + idx, old_moment.operations, new_moment.operations[0])
+        for new_moment, (idx, old_moment) in zip(
+            sequence, enumerate(circuit[index : index + len(sequence)])
+        )
+    ]
```

Comment:
We probably don't need this function based on the current implementation of `insert_ddd_sequences`.